### PR TITLE
feat: [TagInput] support ally aria-*

### PR DESCRIPTION
**Pull request by boomboomchen**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- chore: TagInput 支持 A11y aria

---

🇺🇸 English
- chore: TagInput supports A11y aria

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### CODE REVIEW

File: packages/semi-ui/tagInput/index.tsx [hunk 1]
```diff
@@ -409,6 +421,7 @@ class TagInput extends BaseComponent<TagInputProps, TagInputState> {
                 <div className={wrapperCls}>
                     {this.renderTags()}
                     <Input
+                        aria-label='input'
```
Comment (shijiatongxue) [inline packages/semi-ui/tagInput/index.tsx:424]: 这个 aria-label 不用加，因为原生 input 具有语义性

File: packages/semi-ui/tagInput/index.tsx [hunk 2]
```diff
@@ -236,7 +238,14 @@ class TagInput extends BaseComponent<TagInputProps, TagInputState> {
         });
         if (showClear) {
             return (
-                <div className={clearCls} onClick={e => this.handleClearBtn(e)}>
+                // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```
Comment (shijiatongxue) [inline packages/semi-ui/tagInput/index.tsx:241]: 这里是不是应该加个 onKeyPress 监听一下 enter 键

File: packages/semi-ui/tagInput/index.tsx (hunk 1 above)
Comment (boomboomchen) [reply packages/semi-ui/tagInput/index.tsx:424]: 好像不能移除？会报错Form elements must have labels

File: packages/semi-ui/tagInput/index.tsx (hunk 2 above)
Comment (boomboomchen) [reply packages/semi-ui/tagInput/index.tsx:245]: 已改～

**Pull request merged**